### ISSUE TITLE
Controlling STR-DE885

### ISSUE THREAD

**Issue by Nawor3565** [Ircama/Sony_SLink#8]
Hello! I am trying to use this to change the volume on my Sony STR-DE885 receiver/amp. Unfortunately, I can't get it to work, and I'm wondering if I'm simply wiring it incorrectly. All other sources seem to require the use of a transistor, but the other issues just say that a diode/resistor should be used (but is not necessary). Do you just connect the output pin of the Arduino to the tip of a TRS connector, and the Arduino's ground to the sleeve? 

Thanks!

**Comment by Ircama** [comment]
If I remember well what I did with https://github.com/Ircama/IR2SLink in the past when I tested the circuit, I powered the microcontroller with 3.2V and directly connected one microcontroller pin to S-LINK. I might have used a diode but I have no note, I need to open the device to verify it.

**Comment by Nawor3565** [comment]
Thanks! I hope I didn't damage anything by using 5V... Just to clarify, you connected the Arduino ground to the cable too, right? This is what I imagine the setup would look like

![71281_1_1](https://user-images.githubusercontent.com/19783632/102630661-d301d580-411a-11eb-8f81-598e04349a0d.jpg)

Edit: unless I'm misunderstanding and you mean you opened the amp and soldered directly to the SLINK input on the board?

**Comment by Ircama** [comment]
Yes, the Digispark is powered by the internal standby power supply rail of the amp (5.6V reduced to 5V) and the S-Link is internally hardwired to the S-Link rail following the schematic diagrams of the amp service manual. I confirm that there is a direct wiring as described [here](https://github.com/Ircama/IR2SLink), but I was wrong with the power supply: the Digispark is powered at 5V. [Here](http://ee.bradley.edu/projects/proj2001/sonyntwk/SLink.PDF) you can find a description of the physical layer of the interface.
